test/radius_SUITE.erl

```diff
@@ -57,13 +57,15 @@
 			    secret => <<"secret">>}}},
 	  apps =>
 	      #{default =>
+		    #{'Origin-Host' => <<"dummy.host">>,
+		      procedures =>
 		    #{init         => [#{service => <<"Default">>}],
 		      authenticate => [#{service => <<"RADIUS-Auth">>}],
 		      authorize    => [#{service => <<"RADIUS-Auth">>}],
 		      start        => [#{service => <<"RADIUS-Acct">>}],
```

Comment:
the content of `procedures` needs to be reindent.

Reply:
ok